### PR TITLE
Update index page. Add team page.

### PR DESCRIPTION
**Pull request by stphndxn**

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```
Comment (jamescmartinez) [inline npm-debug.log:1]: I assume committing this was an accident? Confirm and I'll get rid of it.

**Comment by jamescmartinez** [comment]
The new layout doesn't seem to work well on mobile. If we're an iOS community, I think our site should work well there. Thoughts? 

ios simulator screen shot may 15 2015 9 39 46 am
ios simulator screen shot may 15 2015 9 40 00 am

File: npm-debug.log (hunk 1 above)
Comment (stphndxn) [inline npm-debug.log:1]: That's right. Please remove this.

**Comment by stphndxn** [comment]
You're right. I will make changes, test on all iOS devices and create another pull request.

**Comment by jamescmartinez** [comment]
Thanks for being such a good sport, Steve. I set the bar high, but I know you can meet it. :+1:

**Comment by stphndxn** [comment]
Of course! 😜

**Pull request closed without merging**